server/src/domains/auth/dto/authReturn.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from "src/domains/users/entities/user.entity"
+import { TokensReturnDto } from "./tokensReturn.dto"
+
+export class AuthReturnDto extends TokensReturnDto {
+    readonly user: User
```

Comment:
Notice: DTO that returns on Fronetnd should not contain entities, it should only contain other Dtos with sensitive fields like passwords removed